.github/workflows/debian/template/control

```diff
@@ -0,0 +1,8 @@
+Package: avalanche 
+Version: 0.1.0
```

Comment:
Debian package version is hard coded.

Reply:
It is only a place holder. The `build-deb-pkg.sh` script replaces the content of the placeholder with the current version.